cfpb/cfgov-refresh
# URL encode full email share URL | Fixes link targetting

So turns out we weren't urlencoding the full mailto email share URL. 

## Changes

- URL encodes full share via email URL.
- Fixes target and rel noopener noreferrer URL, which was getting double quotes.

<img width="423" alt="screen shot 2018-11-21 at 10 56 04 am" src="https://user-images.githubusercontent.com/704760/48853585-d1b82000-ed7d-11e8-924e-602a16b3c368.png">

## Testing

1. Pull this branch down and run it locally.
2. Visit a page with the sharing molecule, such as blog or find a housing counselor. Click the share via email button and check that the draft email looks correct.
3. Check that FB, Twitter, and LinkedIn URLs are unchanged. Easiest may be to go into master branch and record URLs and then compare to this branch.
